.github/workflows/.lint_python.yml

```diff
@@ -0,0 +1,26 @@
+name: lint_python
+on:
+  pull_request:
+  push:
+  #  branches: [master]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.5, 3.6, 3.7, 3.8]  # , pypy3]
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@master
+      - run: pip install codespell flake8 isort mypy pytest
```

Comment:
I think you should use our requirements files so they don't get out of sync. Further, those support environment markers.

Reply:
These are not runtime requirements.  We do not need end users to install these.  We only need these dependencies at test time.

Reply:
I believe that dev dependencies are typically in `requirements.txt` while end-user ones are in `setup.py`. `pip install pygithub` does not look at `requirements.txt` but only `install_requires` in `setup.py`.

https://packaging.python.org/discussions/install-requires-vs-requirements/

Reply:
Cool!  I have never seen that.  I usually see `requirements_dev.txt` and `requirements.txt` but I am good with the model you suggest.